<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>600</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>InQ: A Sketchbook Application</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/img/inqlogo.webp</normaloff>:/img/img/inqlogo.webp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>Credits</string>
    </property>
    <addaction name="actionAbout_InQ"/>
   </widget>
   <widget class="QMenu" name="menuCustomize">
    <property name="title">
     <string>Customize</string>
    </property>
    <addaction name="actionChange_Resolution"/>
    <addaction name="actionBrush_Color"/>
    <addaction name="actionBrush_Thickness"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionNew_File"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMagnifier"/>
    <addaction name="actionEraser"/>
   </widget>
   <widget class="QMenu" name="menuShapes">
    <property name="title">
     <string>Shapes</string>
    </property>
    <addaction name="actionStraight_Line"/>
    <addaction name="actionCircle"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionTriangle"/>
    <addaction name="actionPentagon"/>
    <addaction name="actionHexagon"/>
    <addaction name="actionFour_Point_Star"/>
    <addaction name="actionArrow"/>
    <addaction name="separator"/>
    <addaction name="actionFloodfill_Color"/>
   </widget>
   <addaction name="menuExit"/>
   <addaction name="menuCustomize"/>
   <addaction name="menuTools"/>
   <addaction name="menuShapes"/>
   <addaction name="menuCredits"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionNew_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave"/>
   <addaction name="actionChange_Resolution"/>
   <addaction name="separator"/>
   <addaction name="actionBrush_Color"/>
   <addaction name="actionBrush_Thickness"/>
   <addaction name="actionFloodfill_Color"/>
   <addaction name="actionEraser"/>
   <addaction name="separator"/>
   <addaction name="actionStraight_Line"/>
   <addaction name="actionCircle"/>
   <addaction name="actionTriangle"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionPentagon"/>
   <addaction name="actionHexagon"/>
   <addaction name="actionFour_Point_Star"/>
   <addaction name="actionArrow"/>
   <addaction name="separator"/>
   <addaction name="actionMagnifier"/>
   <addaction name="separator"/>
   <addaction name="actionFloodfill_Selected_Shape"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_InQ"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAbout_InQ">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/about.png</normaloff>:/img/img/about.png</iconset>
   </property>
   <property name="text">
    <string>About InQ</string>
   </property>
  </action>
  <action name="actionChange_Resolution">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/resolution.png</normaloff>:/img/img/resolution.png</iconset>
   </property>
   <property name="text">
    <string>Change Resolution</string>
   </property>
  </action>
  <action name="actionBrush_Color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/colorpalette4.png</normaloff>:/img/img/colorpalette4.png</iconset>
   </property>
   <property name="text">
    <string>Brush Color</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/exiticon.jpg</normaloff>:/img/img/exiticon.jpg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/save.png</normaloff>:/img/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionMagnifier">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/zoom-in.png</normaloff>:/img/img/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Magnifier</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/eraser.png</normaloff>:/img/img/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionBrush_Thickness">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/brushthickness.png</normaloff>:/img/img/brushthickness.png</iconset>
   </property>
   <property name="text">
    <string>Brush Thickness</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/circle.png</normaloff>:/img/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/rectangle.webp</normaloff>:/img/img/rectangle.webp</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/triangle.png</normaloff>:/img/img/triangle.png</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionFloodfill_Selected_Shape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill Shape</string>
   </property>
  </action>
  <action name="actionFloodfill_Color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/floodfillicon.png</normaloff>:/img/img/floodfillicon.png</iconset>
   </property>
   <property name="text">
    <string>Floodfill Color</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/openfile.webp</normaloff>:/img/img/openfile.webp</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/new.png</normaloff>:/img/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionStraight_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/straightline.png</normaloff>:/img/img/straightline.png</iconset>
   </property>
   <property name="text">
    <string>Straight Line</string>
   </property>
  </action>
  <action name="actionPentagon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/pentagon.png</normaloff>:/img/img/pentagon.png</iconset>
   </property>
   <property name="text">
    <string>Pentagon</string>
   </property>
  </action>
  <action name="actionHexagon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/hexagon.webp</normaloff>:/img/img/hexagon.webp</iconset>
   </property>
   <property name="text">
    <string>Hexagon</string>
   </property>
  </action>
  <action name="actionFour_Point_Star">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/fourpointstar2.png</normaloff>:/img/img/fourpointstar2.png</iconset>
   </property>
   <property name="text">
    <string>Four Point Star</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/arrow (1).png</normaloff>:/img/img/arrow (1).png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
